camel/models/model_factory.py

```diff
@@ -68,6 +69,8 @@ def create(
                 model_class = OpenAIModel
             elif model_platform.is_anthropic and model_type.is_anthropic:
                 model_class = AnthropicModel
+            elif model_type.is_groq:
+                model_class = GroqModel
```

Comment:
set groq as platform, support other model types including mixtral-8x7b, refer to https://console.groq.com/docs/models

Reply:
We can integrate the mistral later (next pr) otherwise, the workload will be doubled (since the support of groq is really bad)? What do you think of it?

Reply:
Hey @Appointat , I think the workload would not be double, just some small change, don't hard code the model name, refer to how Ollama was integrated : camel/models/ollama_model.py